lib/promscrape/config.go

```diff
@@ -83,7 +83,7 @@ func mustInitClusterMemberID() {
 	if err != nil {
 		logger.Fatalf("cannot parse -promscrape.cluster.memberNum=%q: %s", *clusterMemberNum, err)
 	}
-	if n < 0 || n >= *clusterMembersCount {
+	if n < 0 || (*clusterMembersCount != 0 && n >= *clusterMembersCount) {
```

Comment:
this change isn't needed if we set default value of clusterMembersCount to 1, right?

Reply:
Yeah, it depends on whether we allow users to set `clusterMembersCount` to 0, because it's used to be 0.